wolfssl/wolfio.h

```diff
@@ -615,7 +615,82 @@ WOLFSSL_API void wolfSSL_SetIOWriteFlags(WOLFSSL* ssl, int flags);
     WOLFSSL_API   int wolfSSL_SetIO_LwIP(WOLFSSL* ssl, void *pcb,
                                 tcp_recv_fn recv, tcp_sent_fn sent, void *arg);
 #endif
+#ifdef WOLFSSL_ISOTP
```

Comment:
Can you add the `WOLFSSL_ISOTP` build option in the docs?

Reply:
I don't think there is a place for this in the Doxygen docs, documentation PR OK?